Gaijinizer patch/Map059.txt

```diff
@@ -91,9 +93,9 @@ I only had hobbies to observe people ...#MTLed
 あのカルテも、シェルターに記録されてた登録データを
  ハッキングしてプリントアウトしたものなんよ…。
 > CONTEXT: Map059/events/1/pages/0/67/Dialogue/other3
-So I was following the trend of you casually ....
-That chart also registered data recorded in the
-shelter. What hacked and printed out ....#MTLed
+So I ended up following your movements... Those
+medical records are data from the shelter I hacked
+and printed out...#Ron
```

Comment:
Minor flow:
>_**So, I ended up following your movements.
Those medical records are data from the shelter
I hacked and printed out...**_